packages/react/carbon.yml

```diff
@@ -445,6 +460,20 @@ assets:
         name: Storybook
         action: link
         url: https://react.carbondesignsystem.com/?path=/docs/components-search
+    section:
+    name: Section
+    framework: react
+    status: stable
+    type: component
+    platform: web
+    thumbnailPath: ./thumbnails/section.svg
```

Comment:
Does this thumbnail exist? It looks like no. I'd remove `thumbnailPath` if we don't have a thumbnail yet, and work with @kingtraceyj and @jeanservaas to create one.

Reply:
The platform will render a "Preview coming soon" or something if an asset doesn't have a thumbnail yet. It's not the end of the world if assets temporarily don't have thumbnails yet, so I wouldn't let that block this PR. But we're still striving for all publicly-viewable assets to have thumbnails.